.gitignore

```diff
@@ -6,3 +6,4 @@ log.txt
 beta
 tmp
 venv
+env
```

Comment:
Are you sure keeping both env and venv as gitignored is necessary? As long we suggest to run `python3 -m venv env` it's just gonna use `env` to create the virtualenv and nothing more..

Reply:
I'm not sure I got this,  by running `python3 -m venv env`, we create a virtual environment in the `env` folder.

We do not need to commit `env` folder too in our database repo: for this reason, I placed it into .gitignore

Am I misunderstanding your words?

Reply:
Ok, so then why "venv" is also gitignored here?

Reply:
Because `env` and `venv` are two common used names.